metron-stellar/stellar-common/README.md

```diff
@@ -711,6 +713,18 @@ In the core language functions, we support basic functional programming primitiv
     * dateTime - The datetime as a long representing the milliseconds since unix epoch
   * Returns: The current year
 
+### `ZIP`
+  * Description: Zips lists into a single list where the ith element is an list containing the ith items from the constituent lists.
```

Comment:
Well, there is https://en.wikipedia.org/wiki/Convolution_(computer_science), what do you think?